scripts/dependency-check/scan_image.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="$( cd "$SCRIPT_DIR"/../../ && pwd )"
+BUNDLE_DIR="$( mktemp -d -p /tmp )"
+REPORTS_DIR="$REPO_DIR/test_output"
+CACHE_DIR="$HOME/.cache/dependency-check"
+IMAGE_NAME="${1:-"quay.io/signalfx/signalfx-agent-dev:$( "$SCRIPT_DIR"/../current-version )"}"
+
+mkdir -p "$REPORTS_DIR"
+
+cid=$( docker create $IMAGE_NAME true )
+docker export $cid | tar -C "$BUNDLE_DIR" -xf -
+docker rm -fv $cid
+
+docker build -t dependency-check "$SCRIPT_DIR"
+
+set +e
+
+docker run --rm \
+    -v "$BUNDLE_DIR":/bundle \
+    -v "$CACHE_DIR":/usr/share/dependency-check/data \
+    -v "$REPORTS_DIR":/reports \
+    dependency-check \
```

Comment:
Make this the thing in the Makefile.  There is a lot of information encoded in these command line flags that should be more visible.